tests/conftest.py

```diff
@@ -48,3 +49,30 @@ def undo():
             monkeypatch.setattr('click.echo', old_echo)
 
     return Result
+
+
+@pytest.fixture
+def sleep_time(tmpdir):
+    """
+    Returns how long we need to sleep for the filesystem's mtime precision to
+    pick up differences.
+
+    This keeps test fast on systems with high precisions, but makes them pass
+    on those that don't.
+    """
+    tmpfile = tmpdir.join('touch_me')
+
+    def touch_and_mtime():
+        with tmpfile.open('w') as f:
```

Comment:
Apparently, this'll only work the first call, but not the second (since the file will already exist). It doesn't seem to act like `touch` when the file exists.